docs/quick-start-docker.md

```diff
@@ -61,31 +61,34 @@ metrictank_1       | 2016/08/04 11:28:24 [I] carbon-in: listening on :2003/tcp
 metrictank_1       | 2016/08/04 11:28:24 [I] starting listener for metrics and http/debug on :6060
 ```
 
-Once the stack is up, metrictank should be running on port 6060:
+Once the stack is up, metrictank should be running on port 6060. If you're running Docker engine natively, you can connect using `localhost`. If you're using Docker Toolbox (which runs containers inside a VirtualBox VM), the host will be the IP address returned by `docker-machine ip`.
 
 ```
 $ curl http://localhost:6060
 OK
-$ curl http://localhost:6060/cluster
-{"instance":"default","primary":true,"lastChange":"2016-08-02T17:12:25.339785926Z"}
-```
 
-And Grafana will be running on port 3000
+$ curl -s http://localhost:6060/cluster | json_pp
```

Comment:
Feel free to remove the `| json_pp` bit but I think the prettification here helps the readability.